### DISCUSSION TITLE
Change language to Spanish

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3725] [Q&A] [answered]
Hello again

Sorry for the inconvenience, but I think I'm the only Latino and I'm really excited about Owncast
 
Can you help me how to change the default language of the platform to Spanish?

I leave you a screenshot

Greetings and thanks
![owncast 1](https://github.com/owncast/owncast/assets/69484466/f5869998-6c8e-45b7-bb06-a91ca0a17e2f)
![owncast 2](https://github.com/owncast/owncast/assets/69484466/1dae19a8-9968-4960-875f-cdf8936fa03e)
![owncast 3](https://github.com/owncast/owncast/assets/69484466/77ef3c32-b7eb-4daf-b00f-79bf7a3749f8)

**Comment by gabek** [comment, marked as answer]
Please see issue https://github.com/owncast/owncast/issues/1604.

**Reply by fsan2val** [reply]
Thank you very much for answering. I review it

Greetings thank you
